### DISCUSSION TITLE
DataTable filterInput className

### DISCUSSION THREAD

**Discussion by mtsmith** [primefaces/community#332] [PrimeReact] [answered]
I'm trying to create a DataTable with column filters and size="small" and its not automatically applying the p-inputtext-sm class to the filter input controls. So I'm trying to use the pass through options to target the filterInput and that doesn't seem to be working either. Any suggestions?

```
<DataTable
...
pt={{
  filterInput: {
    root: { className: 'p-inputtext-sm' }
  }
}}
/>
```

**Comment by gucal** [comment, marked as answer]
Hi,
This pt must be used for the Column component.

```
<Column
    ...
    pt={{ filterInput: 'p-inputtext-sm' }} 
/>
```
